contrib/google.golang.org/grpc.v12/grpc.go

```diff
@@ -1,3 +1,4 @@
+// Package grpc provides functions to trace the google.golang.org/grpc package v1.2.
```

Comment:
Do we miss an example for `grpc`? maybe we should explain how to use the integration somewhere?

Reply:
Done. Added.